backend/src/aggregator.rs

```diff
@@ -128,6 +116,8 @@ impl Actor for Aggregator {
 pub struct AddNode {
     /// Details of the node being added to the aggregator
     pub node: NodeDetails,
+    /// Genesis `Hash` of the chain the node is being added to.
```

Comment:
Using ```[`Hash`]``` would help navigation in rustdocs.

Suggestion: "Genesis `Hash` of the chain of the node."